client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -62,6 +65,7 @@ class BusinessDetails extends Component {
 		this.extensions = [
 			'facebook-for-woocommerce',
 			'mailchimp-for-woocommerce',
+			'google-ads',
```

Comment:
With the new plugin published, this slug needs to be updated to `kliken-marketing-for-google`